var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -195,45 +214,63 @@ class PyTensorflow(Package, CudaPackage):
     depends_on('py-protobuf@3.0.0', type=('build', 'run'), when='@0.11.0')
     depends_on('py-protobuf@3.0.0b2', type=('build', 'run'), when='@0.7.1:0.10')
     depends_on('py-protobuf@3.0.0a3', type=('build', 'run'), when='@0.6:0.7.0')
-    depends_on('protobuf')
-    depends_on('flatbuffers+python@1.12.0:1.12', type=('build', 'run'), when='@2.4.0:')
+    depends_on('protobuf@:3.12', when='@:2.4')
+    depends_on('protobuf@:3.17')
+    depends_on('flatbuffers+python@1.12:2.9', type=('build', 'run'), when='@2.7:')
```

Comment:
```suggestion
    depends_on('flatbuffers+python@1.12:2', type=('build', 'run'), when='@2.7:')
```
2.10 would also work